### PR TITLE
Adds the ability to eat pie off your face with a fork

### PR DESCRIPTION
**Pull request by adamsong**
# Github documenting your Pull Request

Simple, use fork on face, get forkful of pie
If you are clumsy, you might stab yourself in the eye

# Wiki Documentation
It does the above, gives 1 unit of banana

# Changelog

:cl:  
rscadd: Added the ability to eat pie off your face
imageadd: added fork of pie icon
/:cl:

### CODE REVIEW

**Review by nesisknesis — APPROVED** [review]

File: code/game/objects/items/kitchen.dm [hunk 1]
```diff
@@ -30,6 +30,7 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30)
 	var/datum/reagent/forkload //used to eat omelette
+	var/loaded_egg /// True if the loaded item is an egg, false if its pie
```
Comment (alexkar598) [inline code/game/objects/items/kitchen.dm]: Make it a string instead, not a fucking boolean

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/kitchen.dm (hunk 1 above)
Comment (adamsong) [reply code/game/objects/items/kitchen.dm]: Changed it

File: code/game/objects/items/kitchen.dm [hunk 2]
```diff
@@ -30,7 +30,7 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30)
 	var/datum/reagent/forkload //used to eat omelette
-	var/loaded_egg /// True if the loaded item is an egg, false if its pie
+	var/loaded_food = "nothing" /// The name of the thing on the fork
```
Comment (alexkar598) [inline code/game/objects/items/kitchen.dm:33]: Set it to null not "nothing"

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/kitchen.dm (hunk 2 above)
Comment (adamsong) [reply code/game/objects/items/kitchen.dm:33]: No, then if something goes wrong its says its a fork full of nothing

**Pull request merged**